contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -59,10 +59,10 @@ type T struct {
 	State       StateType
 	Flags       map[FlagType]struct{}
 	CreateTime  time.Time
-	UpdatedTime time.Time // time of the most recent StatusUpdate we've seen from the mesos master
+	UpdatedTime time.Time           // time of the most recent StatusUpdate we've seen from the mesos master
+	Executor    *mesos.ExecutorInfo // readonly
```

Comment:
dislike making this public. encourages callers to mess with its state, which I expressly do NOT want to encourage. would rather see an `ExecutorID() string` func added to task, since it looks like that's the only bit that's needed outside of this package